### PR TITLE
Lg/0.6.26 version

### PR DESCRIPTION
**Pull request by lsgunnlsgunn**
**Overview**
DRAFT release notes/version change.

### CODE REVIEW

**Comment by ililic** [comment]
LGTM … will `link:../interface-spec/index{outfilesuffix}[_{IC} Interface Specification_]` be automatically published?

**Comment by lsgunnlsgunn** [comment]
> LGTM … will `link:../interface-spec/index{outfilesuffix}[_{IC} Interface Specification_]` be automatically published?

When I merge the branch that has it ready, the website will deploy and voila, we'll have the doc in the navigation and accessible to the public.

File: modules/ROOT/pages/download.adoc [hunk 1]
```diff
@@ -38,7 +38,7 @@ To download and install a specific version from a terminal shell:
 
 . Set the `DFX_VERSION` environment variable to the version of the {sdk-short-name} package you want to install as a prefix to `+curl+` command.
 +
-For example, to install version 0.6.25, you would run the following command:
```
Comment (codelemur) [inline modules/ROOT/pages/download.adoc:41]: should we update 0.6.23 [here](https://github.com/dfinity/docs/pull/365/files#diff-5b9761bd3ef159589bc5de48b1f3da118e151215327ef1b79f51a34bd91b9aefR45), too?

Comment (lsgunnlsgunn) [reply modules/ROOT/pages/download.adoc:41]: good catch, eagle-eye!

**Review by hansl — APPROVED** [review]

**Comment by hansl** [comment]
@lsgunnlsgunn Can we add a Known Issue section?

There is a known issue that bls cannot be imported in the current `agent-js` release. This is ONLY a problem when using `@dfinity/agent` directly, it will not happen when using bootstrap, so it should not happen too often. Any developers using the agent directly should use the latest beta version (`@dfinity/agent@0.6.26-beta.1`).

**Pull request merged**